item-spec/json-schema/item.json

```diff
@@ -204,6 +204,21 @@
               "title": "Link title",
               "type": "string"
             }
+          },
+          "$comment": "Link with relationship `self` must be absolute URI",
+          "if": {
+            "properties": {
+              "rel": {
+                "const": "self"
+              }
+            }
+          },
+          "then": {
+            "properties": {
+              "href": {
+                "format": "iri"
```

Comment:
What about local (filesystem) catalogs? E.g. with a self link of `/path/to/my/catalog.json`?

Reply:
Shouldn't that be using the file protocol? file://path/to/my/...?

Reply:
Most tooling that I've seen _won't_ do that, e.g. **pystac**.

Reply:
I'd say then pystac is generating invalid STAC. The STAC Spec in 1.0 says explicitly "Absolute URL" and /path/to/my/catalog.json is not an absolute URL, it's just an absolute path.

Reply:
Then we need to fix the spec, whick pretty much always talks about URLs. That might actually require a 2.0 though, because some tooling may now expect URLs (such as STAC Browser).